frontend/projects/upgrade/src/app/core/http-interceptors/http-error.interceptor.ts

```diff
@@ -32,7 +32,9 @@ export class HttpErrorInterceptor implements HttpInterceptor {
           this.authService.authLogout();
         }
         this.openPopup(err);
-        return EMPTY; // returning EMPTY instead of throwError as Error is handled using snacker here itself
+
+        // re-throw to allow the error to be caught by the calling code
+        return throwError(() => new Error(JSON.stringify(err)));
```

Comment:
Why not just `throw error`?

Reply:
I'm sorry, what is the code suggestion to try?

Reply:
ok I see just `throw err` does work, let me fix the tests.

Reply:
fixed @VivekFitkariwala